### PR TITLE
add search in my Skill section

### LINKED ISSUE #185: Search bar in "my skillz"
Refered to the user feedbacks in Miro, we should have a search bar in "my Skillz" section

### PR DESCRIPTION
**Pull request by MailyLehoux**
resolve #185

### CODE REVIEW

**Comment by ZenikAdrien** [comment]
OK sur la feature (testée en local) - un petit commentaire sur le code

File: src/pages/skills/[context]/[category]/index.tsx [hunk 1]
```diff
@@ -56,26 +61,41 @@ const ListSkills = () => {
...
+            x: skill.skillLevel,
+            y: skill.desireLevel,
+            weight: 65,
+            labels: [skill.name],
+            name: skill.name,
+          }))
+      );
+    }
+  }),
+    [search];
```
Comment (benjaminbrgs) [inline src/pages/skills/[context]/[category]/index.tsx:91]:
```suggestion
  useEffect(() => {
    if (skillsData && skillsData.length) {
      setRadarData(
        skillsData
          .filter(
            (skill) =>
              (skill.skillLevel > 0 || skill.skillLevel) &&
              (skill.desireLevel > 0 || skill.desireLevel)
          )
          .map((skill) => ({
            x: skill.skillLevel,
            y: skill.desireLevel,
            weight: 65,
            labels: [skill.name],
            name: skill.name,
          }))
      );
    } else {
        setRadarData([]);
    }
  }, [search, !loading]);
```

File: src/pages/skills/[context]/[category]/index.tsx [hunk 2]
```diff
@@ -56,26 +61,41 @@ const ListSkills = () => {
...
+
   useEffect(() => {
     if (!skillsData || skillsData.length <= 0) {
       setRadarData([]);
       return;
     }
```
Comment (benjaminbrgs) [inline src/pages/skills/[context]/[category]/index.tsx:97]:
```suggestion
```

**Review by benjaminbrgs — COMMENTED** [review]
Some suggestions because useEffect is misconfigured and produces an infinite render

**Review by benjaminbrgs — APPROVED** [review]

**Comment by MailyLehoux** [comment]
Thanks you both for your reviews. It's ready to merge

**Pull request merged**